chrysler_pacifica_ehybrid_s_2019/docker-compose.yml

```diff
@@ -0,0 +1,121 @@
+#  Copyright (C) 2018-2019 LEIDOS.
+# 
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+# 
+#  http://www.apache.org/licenses/LICENSE-2.0
+# 
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+# Docker Compose Spec Version
+version: '2'
+
+services:
+  roscore:
+    image: usdotfhwastol/carma-base:2.9.0
+    network_mode: host
+    container_name: roscore
+    volumes_from: 
+      - container:carma-config:ro
+    environment:
+      - ROS_IP=192.168.88.10
+    volumes:
+      - /opt/carma/.ros:/home/carma/.ros
+    restart: always
+    command: roscore
+    
+  platform:
+    image: usdotfhwastol/carma:2.9.0
+    network_mode: host
+    container_name: platform
+    volumes_from: 
+      - container:carma-config:ro
+    environment:
+      - ROS_IP=192.168.88.10
+    volumes:
+      - /opt/carma/logs:/opt/carma/logs
+      - /opt/carma/.ros:/home/carma/.ros
+      - /opt/carma/vehicle/HostVehicleParams.yaml:/opt/carma/params/HostVehicleParams.yaml
+    command: wait-for-it.sh localhost:11311 -- roslaunch carma carma_docker.launch
+
+  cohda_dsrc_driver:
+    image: usdotfhwastol/carma-cohda-dsrc-driver:1.0.0
+    container_name: carma-cohda-dsrc-driver
+    network_mode: host
+    volumes_from: 
+      - container:carma-config:ro
+    environment:
+      - ROS_IP=192.168.88.10
+      - ROS_NAMESPACE=hardware_interface
+    volumes:
+      - /opt/carma/logs:/opt/carma/logs
+      - /opt/carma/.ros:/home/carma/.ros
+      - /opt/carma/vehicle/HostVehicleParams.yaml:/opt/carma/params/HostVehicleParams.yaml
+    command: wait-for-it.sh localhost:11311 -- roslaunch dsrc_driver dsrc_node.launch
+
+  pacmod_controller_driver:
```

Comment:
Is that still pacmod driver for Pacifica?

Reply:
No it is not. This PR is just to update the urdf file per CAR-419. I copied the other files over for convenience but there are still issues with them.